blueman/Functions.py

```diff
@@ -45,13 +45,11 @@
 gi.require_version("Gdk", "3.0")
 from gi.repository import Gtk
 from gi.repository import Gdk
-from gi.repository import GdkPixbuf
```

Comment:
I just noticed that this is still used in ``create_menuitem`` so please back this out before merging.